FWCore/Framework/interface/ESProducer.h

```diff
@@ -4,7 +4,7 @@
 //
 // Package:     Framework
 // Class  :     ESProducer
-// 
```

Comment:
Were there any functional changes to this file?

Reply:
I think the answer is no.  I'll be more careful next time.